common/test/acceptance/pages/studio/index.py

```diff
@@ -86,16 +86,22 @@ def submit_new_library_form(self):
 
     def list_libraries(self):
         """
-        List all the libraries found on the page's list of libraries.
+        Click the tab to display the available libraries, and return detail of them.
         """
         # Workaround Selenium/Firefox bug: `.text` property is broken on invisible elements
         self.q(css='#course-index-tabs .libraries-tab a').click()
+        if self.q(css='.list-notices.libraries-tab').present:
+            # No libraries are available.
+            self.wait_for_element_visibility('.libraries-tab .new-library-button', "Switch to library tab")
+            return []
+        self.q(css='#course-index-tabs .libraries-tab a').click()
```

Comment:
Are you intentionally clicking it twice now, or should the click() on line 92 be removed?